### ISSUE TITLE
Can I make an Among Us game by python? (Simple one)

### ISSUE THREAD

**Issue by icrushhaileesteinfeld** [geekcomputers/Python#1703]

**Comment by SarwinR** [comment]
Yes you can but it might take much more time than using a game engine like Unity or Unreal Engine. 
Fun Fact: Among Us was developed using the Unity engine.

**Comment by TSaiPradyumna** [comment]
> _No description provided._

hello
have you asked the question?
that any one can create a game?
if anked and is ya a little thing not big

**Comment by AliThePassionate** [comment]
what we make among us game using python?

**Comment by NitkarshChourasia** [comment]
> what we make among us game using python?

here I am ! the maintainer of the repository. Invite me for a pair programming, too.
